modules/authentication/src/handlers/local.ts

```diff
@@ -787,21 +785,13 @@ export class LocalHandlers implements IAuthenticationStrategy {
       email: email,
     });
     if (isNil(user)) throw new GrpcError(status.NOT_FOUND, 'User not found');
-    if (user.isVerified) return 'User is already verified';
+    if (user.isVerified)
+      throw new GrpcError(status.FAILED_PRECONDITION, 'User already exists');
```

Comment:
`User is already verified` was fine. I only requested that the return be changed to a throw.